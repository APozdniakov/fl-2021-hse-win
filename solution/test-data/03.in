states=[q1]
alpha=[a,b,c,d]
initial=[q1]
accepting=[q1]
trans=[q1>a>q1,q1>c>q1,q1>b>q1,q1>d>q1]
