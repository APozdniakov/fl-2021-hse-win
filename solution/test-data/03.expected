After 0-th step:
R [0] [0] : a|c|b|d
After 1-th step:
R [0] [0] : (a|c|b|d|eps)(a|c|b|d|eps)*(a|c|b|d|eps)|(a|c|b|d|eps)
FINAL ANSWER: (a|c|b|d|eps)(a|c|b|d|eps)*(a|c|b|d|eps)|(a|c|b|d|eps)
